topics/contributing/tutorials/github-contribution/tutorial.md

```diff
@@ -79,6 +101,29 @@ To modify the content of the repository, you need a copy of it on your computer.
 
 ![Explanation of the cloning](../../images/PR_schema_02.png "Cloning a repository"){: width="900px"}
 
+<div class="Desktop" markdown="1">
+
+> <hands-on-title>Clone the GitHub repository</hands-on-title>
+>
+> 1. Get the URL of your fork. You can find this on the main page of your fork under the green button:
+>    1. Click on **Code** (right)
+>
+>       ![Get fork URL](../../images/clone_code_button.png)
+>
+>    2. Select **Open with Github Desktop**
+>
+>       Your computer might ask if it's ok to open the website Select *Allow* if so.
+>     
+>       ![Clone a Repository](../../images/clone_a_repo.png)
+>
+>    2. If you are happy with the location of the folder, select **Clone**
+>       This may take some time the first time (as the GTN repository is quite large!), but eventually the folder will be in your computer.
+{: .hands_on}
+
+</div>
+
+<div class="Command-line" markdown="1">
```

Comment:
It is not clear to me when reading the [preview on GitHub](https://github.com/galaxyproject/training-material/blob/d48c0577a73157add00b820e59cd1960a336ca46/topics/contributing/tutorials/github-contribution/tutorial.md#create-a-fork-of-this-repository-on-github) if you are giving the learner a choice of GitHub Desktop or command line, with alternative instructions depending which they choose. The clone instructions describe using the Terminal. I didn't see any GitHub Desktop screenshots (or maybe any other instructions specific to Desktop) until the review section, "Make any requested changes". Am I missing something?